<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AssetEditionDialog</class>
 <widget class="QDialog" name="AssetEditionDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>807</width>
    <height>697</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true">Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QGroupBox" name="globalParamsBox">
       <property name="title">
        <string>Global parameters</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_4">
        <item row="0" column="0">
         <layout class="QGridLayout" name="globalParamsGrid">
          <item row="1" column="0">
           <widget class="QLabel" name="posLabel">
            <property name="text">
             <string>Position</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0" colspan="3">
           <widget class="QLabel" name="nameLabel">
            <property name="text">
             <string>Name :</string>
            </property>
           </widget>
          </item>
          <item row="5" column="0" colspan="3">
           <widget class="QLabel" name="scaleLabel">
            <property name="text">
             <string>Scale :</string>
            </property>
           </widget>
          </item>
          <item row="2" column="2">
           <widget class="QLabel" name="posYLabel">
            <property name="text">
             <string>Y</string>
            </property>
           </widget>
          </item>
          <item row="5" column="3">
           <widget class="QDoubleSpinBox" name="scaleSpinBox">
            <property name="locale">
             <locale language="English" country="UnitedStates"/>
            </property>
            <property name="suffix">
             <string notr="true">%</string>
            </property>
            <property name="decimals">
             <number>1</number>
            </property>
            <property name="minimum">
             <double>5.000000000000000</double>
            </property>
            <property name="maximum">
             <double>500.000000000000000</double>
            </property>
            <property name="singleStep">
             <double>5.000000000000000</double>
            </property>
            <property name="value">
             <double>100.000000000000000</double>
            </property>
           </widget>
          </item>
          <item row="1" column="2">
           <widget class="QLabel" name="posXLabel">
            <property name="text">
             <string>X</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1" rowspan="3">
           <widget class="Line" name="posVLine">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
           </widget>
          </item>
          <item row="3" column="3">
           <widget class="QDoubleSpinBox" name="posZSpinBox">
            <property name="locale">
             <locale language="English" country="UnitedStates"/>
            </property>
            <property name="suffix">
             <string notr="true"> m</string>
            </property>
            <property name="minimum">
             <double>-100.000000000000000</double>
            </property>
            <property name="maximum">
             <double>100.000000000000000</double>
            </property>
           </widget>
          </item>
          <item row="3" column="2">
           <widget class="QLabel" name="posZLabel">
            <property name="text">
             <string>Z</string>
            </property>
           </widget>
          </item>
          <item row="1" column="3">
           <widget class="QDoubleSpinBox" name="posXSpinBox">
            <property name="locale">
             <locale language="English" country="UnitedStates"/>
            </property>
            <property name="suffix">
             <string notr="true"> m</string>
            </property>
            <property name="minimum">
             <double>-100.000000000000000</double>
            </property>
            <property name="maximum">
             <double>100.000000000000000</double>
            </property>
           </widget>
          </item>
          <item row="0" column="3">
           <widget class="QLineEdit" name="nameLineEdit">
            <property name="text">
             <string notr="true"/>
            </property>
           </widget>
          </item>
          <item row="2" column="3">
           <widget class="QDoubleSpinBox" name="posYSpinBox">
            <property name="locale">
             <locale language="English" country="UnitedStates"/>
            </property>
            <property name="suffix">
             <string notr="true"> m</string>
            </property>
            <property name="minimum">
             <double>-100.000000000000000</double>
            </property>
            <property name="maximum">
             <double>100.000000000000000</double>
            </property>
           </widget>
          </item>
          <item row="4" column="0" colspan="3">
           <widget class="QLabel" name="rotationLabel">
            <property name="text">
             <string>Rotation :</string>
            </property>
           </widget>
          </item>
          <item row="4" column="3">
           <widget class="QDoubleSpinBox" name="rotationSpinBox">
            <property name="suffix">
             <string notr="true">Â°</string>
            </property>
            <property name="decimals">
             <number>1</number>
            </property>
            <property name="minimum">
             <double>-360.000000000000000</double>
            </property>
            <property name="maximum">
             <double>360.000000000000000</double>
            </property>
            <property name="singleStep">
             <double>5.000000000000000</double>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QGroupBox" name="customParamsBox">
       <property name="title">
        <string>Custom parameters</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_6">
        <item row="0" column="0">
         <layout class="QGridLayout" name="customParamsGrid"/>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>AssetEditionDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>AssetEditionDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>nameLineEdit</sender>
   <signal>textChanged(QString)</signal>
   <receiver>AssetEditionDialog</receiver>
   <slot>setWindowTitle(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>435</x>
     <y>62</y>
    </hint>
    <hint type="destinationlabel">
     <x>403</x>
     <y>348</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
